### PR TITLE
jmap: add more information link

### PR DESCRIPTION
**Pull request by BillLucky**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/jmap.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # jmap
 
 > Java Memory Map Tool.
+> More information: <https://docs.oracle.com/javase/7/docs/technotes/tools/share/jmap.html>.
```
Comment (navarroaxel) [inline pages/common/jmap.md:4]: We prefer Manned in general, and I think it fits for this page:
```suggestion
> More information: <https://manned.org/jmap>.
```

Comment (marchersimon) [reply pages/common/jmap.md:4]: Both man pages contain the exact same content and I assume `jmap` was made by oracle, so I'd say we should keep the old one.

Comment (sbrl) [reply pages/common/jmap.md:4]: Yeah, `jmap` is part of Java, so unfortunately the Oracle website is the preferred source here.

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — COMMENTED** [review]
Thanks @BillLucky

**Review by marchersimon — APPROVED** [review]

**Pull request merged**